Testing/Unit/PowerShell/Orchestrator/Invoke-ScubaConfig.Tests.ps1

```diff
@@ -0,0 +1,136 @@
+$OrchestratorPath = '../../../../PowerShell/ScubaGear/Modules/Orchestrator.psm1'
+$ScubaConfigPath = '../../../../PowerShell/ScubaGear/Modules/ScubaConfig/ScubaConfig.psm1'
+$ConnectionPath = '../../../../PowerShell/ScubaGear/Modules/Connection/Connection.psm1'
+
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $OrchestratorPath) -Function 'Invoke-SCuBA' -Force
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $ScubaConfigPath)
+
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $ConnectionPath) -Function Disconnect-SCuBATenant
+
+# This construct did not work
+# . (Join-Path -Path $PSScriptRoot -ChildPath '../PsTestUtils.ps1')
+# Using this instead
+$TestUtilsPath = "../PsTestUtils.psm1"
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath $TestUtilsPath)
+
+
+InModuleScope Orchestrator {
+    Describe -Tag 'Orchestrator' -Name 'Invoke-Scuba with Config' {
+        BeforeAll {
+            function Remove-Resources {}
+            Mock -ModuleName Orchestrator Remove-Resources {}
+            function Import-Resources {}
+            Mock -ModuleName Orchestrator Import-Resources {}
+            function Invoke-Connection {}
+            Mock -ModuleName Orchestrator Invoke-Connection { @() }
+            function Get-TenantDetail {}
+            Mock -ModuleName Orchestrator Get-TenantDetail { '{"DisplayName": "displayName"}' }
+            function Invoke-ProviderList {}
+            Mock -ModuleName Orchestrator Invoke-ProviderList {}
+            function Invoke-RunRego {}
+            Mock -ModuleName Orchestrator Invoke-RunRego {}
+
+            Mock -ModuleName Orchestrator Invoke-ReportCreation {}
+            function Disconnect-SCuBATenant {}
+            Mock -ModuleName Orchestrator Disconnect-SCuBATenant {}
+
+            Mock -CommandName New-Item {}
+            Mock -CommandName Copy-Item {}
+        }
+        Context 'Test Compare-Hash utility used by Invoke-Scuba -ConfigFilePath tests' {
```

Comment:
You data driven approach here as well.

Reply:
These tests are no longer needed due to function being eliminated